HYUNSANG_OH/app.js

```diff
@@ -75,45 +80,103 @@ app.get("/userPost/:userId", async (req, res) => {
     `,
     [userId]
   );
-  const result = userPost.map((userPost) => ({
-    ...userPost,
-    postings: JSON.parse(userPost.postings),
-  }));
-  res.status(200).json({ data: result });
+  res.status(200).json({ data: userPost });
 });
 
 //회원가입
 app.post("/signup", async (req, res, next) => {
   const { name, email, password, profile_image } = req.body;
+
+  const [check] = await appDataSource.query(
+    `
+    SELECT EXISTS
+    (SELECT * FROM users
+    WHERE email = ?)
+    AS 'user';
+    `,
+    [email]
+  );
+  if (check.user == 1) {
```

Comment:
```suggestion
  if (check.user) {
```
단순 truthy or falsy 판단이라면 다음과 같이 간결하게 작성할 수 있을 것 같습니다!